.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,68 @@
+name: Nightly Release
+
+on:
+  workflow_dispatch:  # Manual trigger
+  schedule:
+  - cron: '0 5 * * *' # 5 AM UTC = Midnight EST
+
+jobs:
+  nightly:
+    if: ${{ github.repository == 'shipwright-io/triggers' }}
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write  # To be able to get OIDC ID token to sign images.
+      contents: write  # To be able to update releases.
+      packages: write  # To be able to push images and signatures.
+
+    env:
+      IMAGE_HOST: ghcr.io
+      IMAGE_NAMESPACE: ${{ github.repository }}
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.19.x'
+        cache: true
+        check-latest: true
+
+    # Install tools
+    - uses: ko-build/setup-ko@v0.6
+      with:
+        version: v0.13.0
```

Comment:
Please make use of the [common setup action](https://github.com/shipwright-io/triggers/blob/ab1363587736111e726e69394a246d0a69320820/.github/actions/setup/action.yaml), a usage example is the following:

https://github.com/shipwright-io/triggers/blob/ab1363587736111e726e69394a246d0a69320820/.github/workflows/test.yaml#L20

Reply:
fixed